tests/Fixer/ControlStructure/YodaStyleFixerTest.php

```diff
@@ -890,11 +874,6 @@ function a() {
                 'less_and_greater' => false,
             ],
         ];
-
-        yield [
-            '<?php $a ??= 4 === $b ? 2 : 3;',
-            '<?php $a ??= $b === 4 ? 2 : 3;',
-        ];
```

Comment:
I don't see a second case with these values, was it removed correctly?

Reply:
Yes, it hides here: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.62.0/tests/Fixer/ControlStructure/YodaStyleFixerTest.php#L778